src/lib/controllers/configuration-manager/generators/generate-ec-keypair.ts

```diff
@@ -0,0 +1,17 @@
+import { ECKeyPairOptions, generateKeyPairSync } from 'crypto';
+import { base64 } from '../filters';
+
+export async function generateECKeypair(): Promise<string> {
+	const options = {
+		namedCurve: 'prime256v1',
+		publicKeyEncoding: {
+			type: 'spki',
+			format: 'pem',
+		},
+		privateKeyEncoding: {
+			type: 'pkcs8',
+			format: 'pem',
+		},
+	} as ECKeyPairOptions<'pem', 'pem'>;
```

Comment:
Does this not already conform to `ECKeyPairOptions`?

Reply:
It does, but we need to declare the specific PubF and PrivF format of the type.